Tests/StreamChatTests/Models/Attachments/AttachmentFileType_Tests.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+import XCTest
+
+final class AttachmentFileType_Tests: XCTestCase {
+    // MARK: - isAudio
+
+    func test_isAudio_returnsTrueForExpectedValues() {
+        let expectedValues: Set<AttachmentFileType> = [
+            .mp3,
+            .mp4,
+            .wav,
+            .ogg,
+            .m4a
+        ]
+
+        AttachmentFileType.allCases.forEach { subject in
+            XCTAssertEqual(subject.isAudio, expectedValues.contains(subject))
+        }
+    }
```

Comment:
Worth testing the `returnsFalse` case as well. We should put an array of some types which are not Audio, and it should returns false

Reply:
But we have expected values. It feels unnecessary and it will require to update the test even when isAudio hasn't changed

Reply:
The thing here is that we are not testing all the branches. 
```swift
public var isAudio: Bool {
        switch self {
        case .mp3, .mp4, .wav, .ogg, .m4a:
            return true
        default:
            return false <--- Not tested
        }
    }
```

If in the future someone changes the implementation to this:
```swift
public var isAudio: Bool {
        return true
    }
```

The test would still pass

Reply:
Valid point. If you don't mind i'm going to update this test on the next PR

Reply:
Actually, just realised it. Due to the assert used here
```
XCTAssertEqual(subject.isAudio, expectedValues.contains(subject))
```
any value that is not in the expectedValues should return false. This should be sufficient to cover all cases (including the one you mentioned above). I'll update the name on the next PR to be clearer